figures/4.figure4/scripts/figure4.r

```diff
@@ -0,0 +1,903 @@
+    secreted_proteins == "GDNF [NSU]" ~ "GDNF",
+    secreted_proteins == "C5 [NSU]" ~ "C5",
+    secreted_proteins == "IL-1 RA" ~ "IL-1 RA",
+    secreted_proteins == "IL-17D [NSU]" ~ "IL-17D",
+    secreted_proteins == "IL-27 [NSU]" ~ "IL-27",
+    secreted_proteins == "Osteopontin (OPN) [NSU]" ~ "Osteopontin",
+    secreted_proteins == "FGF-9 [NSU]" ~ "FGF-9",
+    secreted_proteins == "BAFF [NSU]" ~ "BAFF",
+    secreted_proteins == "TGF-beta 3 [NSU]" ~ "TGF-beta 3",
+    secreted_proteins == "EGF [NSU]" ~ "EGF",
+    secreted_proteins == "IL-5 [NSU]" ~ "IL-5",
+    secreted_proteins == "FGF-7 (KGF) [NSU]" ~ "FGF-7 (KGF)",
+    secreted_proteins == "APRIL [NSU]" ~ "APRIL",
+    secreted_proteins == "WISP-1 (CCN4) [NSU]" ~ "WISP-1 (CCN4)",
+    secreted_proteins == "CCL22 [NSU]" ~ "CCL22",
+    secreted_proteins == "FGF-19 [NSU]" ~ "FGF-19",
+    secreted_proteins == "M-CSF [NSU]" ~ "M-CSF",
+    secreted_proteins == "CXCL10 [NSU]" ~ "CXCL10",
+    secreted_proteins == "TGF-beta 1 (total) [NSU]" ~ "TGF-beta 1 ",
+    secreted_proteins == "Tie-2 [NSU]" ~ "Tie-2",
+    secreted_proteins == "TGF-beta 1 (LAP domain in precursor) [NSU]" ~ "TGF-beta 1",
+    secreted_proteins == "FGFR3 (IIIc) [NSU]" ~ "FGFR3 (IIIc)",
+    secreted_proteins == "AITRL (GITR Ligand) [NSU]" ~ "AITRL (GITR Ligand)",
+    secreted_proteins == "Amphiregulin [NSU]" ~ "Amphiregulin",
+    secreted_proteins == "BMP4 [NSU]" ~ "BMP4",
+    secreted_proteins == "G-CSF [NSU]" ~ "G-CSF",
+    secreted_proteins == "TGF-beta 2 [NSU]" ~ "TGF-beta 2",
+    secreted_proteins == "IL-6 R alpha [NSU]" ~ "IL-6 R alpha",
+    secreted_proteins == "BMP6 [NSU]" ~ "BMP6",
+    secreted_proteins == "NGF beta [NSU]" ~ "NGF beta",
+    secreted_proteins == "IL-1 R1 [NSU]" ~ "IL-1 R1",
+    secreted_proteins == "MMP-10 [NSU]" ~ "MMP-10",
+    secreted_proteins == "IL-17F [NSU]" ~ "IL-17F",
+    secreted_proteins == "IL-18 [NSU]" ~ "IL-18",
+    secreted_proteins == "CXCL6 [NSU]" ~ "CXCL6",
+    secreted_proteins == "IL-6 [NSU]" ~ "IL-6",
+    secreted_proteins == "CXCL1 [NSU]" ~ "CXCL1",
+    secreted_proteins == "VEGF-C [NSU]" ~ "VEGF-C",
+    secreted_proteins == "Resistin [NSU]" ~ "Resistin",
+    secreted_proteins == "EMMPRIN [NSU]" ~ "EMMPRIN",
+    secreted_proteins == "IFN gamma [NSU]" ~ "IFN gamma",
+    secreted_proteins == "CCL20 [NSU]" ~ "CCL20",
+    secreted_proteins == "CRP [NSU]" ~ "CRP",
+    secreted_proteins == "VCAM-1 [NSU]" ~ "VCAM-1",
+    secreted_proteins == "Cytochrome C [NSU]" ~ "Cytochrome C",
+    secreted_proteins == "BMP3 [NSU]" ~ "BMP3",
+    secreted_proteins == "IL-24 [NSU]" ~ "IL-24",
+    secreted_proteins == "IL-28A [NSU]" ~ "IL-28A",
+    secreted_proteins == "CCL17 [NSU]" ~ "CCL17",
+    secreted_proteins == "BMP2 [NSU]" ~ "BMP2",
+    secreted_proteins == "CD27L [NSU]" ~ "CD27L",
+    secreted_proteins == "NRG1 beta 1 [NSU]" ~ "NRG1 beta 1",
+    secreted_proteins == "IL-11 [NSU]" ~ "IL-11",
+    TRUE ~ secreted_proteins
+  ))
```

Comment:
Is there a pattern which would work here instead of the multi-line cases (for ex. regex, or similar)?

Reply:
I thought about it and each one is a little bit different so I had to do it manually.